### ISSUE TITLE
stdout is blocked by screenshotPath

### ISSUE THREAD

**Issue by monolithed** [webdriverio/webdriverio#1760]
## The problem

**screencast**: http://g.recordit.co/67xMC8JcyX.gif
**project**: [1752.zip](https://github.com/webdriverio/webdriverio/files/639093/1752.zip)

The problem is being solved by removing `screenshotPath`!

## Environment

* WebdriverIO version: latest
* Node.js version: 7
* Mode: testrunner
* Type: synchronous

**Comment by wvankuipers** [comment]
I think this is a duplicate of/related to #1591

**Comment by monolithed** [comment]
Duplicate of https://github.com/webdriverio/webdriverio/issues/1591
